index.html

```diff
@@ -356,20 +356,55 @@ <h2>Coordination</h2>
         <p>Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/Consortium/Process/#WGCharter">W3C Process Document</a>:</p>
 
         <section>
-          <h3 id="w3c-coordination">W3C Groups</h3>
-          <dl>
-            <dt><a href="">Decentralized Identifier Working Group</a></dt>
-            <dd><i class="todo">[specific nature of liaison]</i></dd>
-          </dl>
+            <h3 id="w3c-coordination">W3C Groups</h3>
+            <dl>
+                <dt><a href="https://www.w3.org/RCH/WG/">RDF Canonicalization and Hashing Working Group</a></dt>
+                <dd>
+                    To synchronize on canonicalization output expression mechanisms that might be used by the Linked Data Integrity specification.
```

Comment:
per Ivan's suggestion:
```suggestion
                    To synchronize on canonicalization output expression mechanisms that might be used by the VC Linked Data Integrity specification.
```